api/NESSIE-SPEC-2-0.md

```diff
@@ -0,0 +1,45 @@
+# Nessie Specification
+
+Nessie Specifications define the behaviour of Nessie Servers. The same behaviour can be expected to be observable
+via all supported API (unless noted otherwise).
+
+[Semver](https://semver.org/spec/v2.0.0.html) principles are followed in bumping the Nessie spec version.
+For example, a major behaviour change will result in incrementing the major version number.
+
+The patch version number will be incremented only on minute spec corrections that do not significantly alter the 
+server's behaviour.
+
+If a server returns a specific spec version via `NessieConfiguration.getSpecVersion()` it MUST comply with the
+specified behaviour (see sections below).
+
+Servers SHOULD return the spec version that defines its behaviour most completely.
+
+A server SHOULD return `null` from `NessieConfiguration.getSpecVersion()`, when it behaves in a way that is not
+compatible with any known spec version.
+
+Servers MAY show behaviours that are not covered by the spec, as long as those behaviours do not contradict the
+well-specified behaviours.
+
+Refer to the Nessie API documentation (under the `model` module) for the meaning of Nessie-specific terms.
```

Comment:
Beware that this PR has moved this doc one level up (it's not under the `model` module anymore).

Reply:
but `model` is a sibling of this file now.

Reply:
That was my point... there is no Nessie API documentation under the `model` module anymore, is there? You could change to stg like:

```suggestion
Refer to the [Nessie API documentation](./README.md) for the meaning of Nessie-specific terms.
```